dashboard/test/ui/features/submittableEyes.feature

```diff
@@ -15,3 +15,19 @@ Scenario: Submittable level
   And I wait until element ".unsubmitButton" is visible
   And I see no difference for "submitted puzzle"
   And I close my eyes
+
+Scenario: Lockable level
+  When I open my eyes to test "lockable level"
+  Then I am on "http://studio.code.org/s/allthethings"
+  And I see no difference for "course overview"
```

Comment:
This is a little funky (and may be specific to localhost environments) but the first time we take a snapshot (i.e. see no difference ...) it resizes the window. If that happens after scrolling our lockable stage into view, the resize results in the stage again being out of view. Solution was to take a snapshot (which results in a resize), scroll into view, then take a second snapshot.